packages/core/js-sdk/src/client.ts

```diff
@@ -30,6 +37,8 @@ const normalizeRequest = (
   return {
     ...init,
     headers,
+    // TODO: Setting this to "include" poses some security risks, as it will send cookies to any domain. We should consider making this configurable.
+    credentials: config.auth?.type === "session" ? "include" : "omit",
```

Comment:
theoratically it wont, the browser manage that depending on the cookie configuration I believe, and on the cookie you can set the domain

Reply:
@adrien2p this is what happens now, I run the admin and BE on different ports, and with `include` it happily sends it to the BE (which is a different domain, so to speak).

Reply:
yes because the configuration locally is loose, but not in production

Reply:
here is more info https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent

Reply:
@adrien2p yeah I understand if you configure it correctly it won't but what I am saying is it isn't currently, and this is  what poses a security risk in this case, so I was thinking we maybe let the user decide if they want to allow it or not. As long as they don't meddle with the session config, and we don't mess it up it's ok, but it is not unheard of to misconfigure things 😄

Reply:
But since the cookie is configured on the BE and we manage the configuration, couldn't we throw if we dont have the whole required config when starting ? It would eliminate security issue no?

Reply:
Oli told be that it might default to the domain it was set from, so maybe just changing the port is not enough to test ahah

Reply:
@adrien2p the default config is `same-origin`, so the cookie would be sent, but only if it is the same origin as the web. So `localhost:9000` and `localhost:5173` are considered different origins, and `same-origin` doesn't work, but `include` does.

Anyway I'll look into it more, but even if we control the configuration it doesn't mean we can't have a bug that would expose the cookie to others by mistake.